internal/test/rego_test.go

```diff
@@ -16,8 +15,11 @@ import (
 	"github.com/styrainc/regal/pkg/builtins"
 )
 
+// nolint:paralleltest,tparallel
 func TestRunRegoUnitTests(t *testing.T) {
-	t.Parallel()
+	// Temporary until this requirement is removed from OPA
+	// Once removed, re-add t.Parallel()
+	t.Setenv("EXPERIMENTAL_GENERAL_RULE_REFS", "true")
```

Comment:
suggestion: prefix comment with TODO to make it a clearer reminder?

Reply:
Will add, thanks!